packages/suite/src/reducers/suite/suiteReducer.ts

```diff
@@ -12,10 +12,12 @@ import { ensureLocale } from '@suite-utils/l10n';
 import { getNumberFromPixelString } from '@trezor/utils';
 import type { OAuthServerEnvironment } from '@suite-types/metadata';
 import type { InvityServerEnvironment } from '@wallet-types/invity';
+import type { CoinjoinServerEnvironment } from '@suite-common/wallet-types';
 
 export interface DebugModeOptions {
     invityServerEnvironment?: InvityServerEnvironment;
     oauthServerEnvironment?: OAuthServerEnvironment;
+    coinjoinServerEnvironment?: CoinjoinServerEnvironment;
```

Comment:
coinjoinRegtestServerEnvironment?

Reply:
imho `coinjoinServerEnvironment` is good enough.

Reply:
I would imagine adding some "dev" environment for testnet and mainnet later, so `coinjoinServerEnvironment` sounds good to me too

Reply:
That seems unnecessarily complicated to me - wouldn't it be sufficient to set it up for regtest/testnet/mainnet all at once?

Reply:
but different networks uses different addresses no?
coinjoinRegtestServerEnvironment != coinjoinTestnetServerEnvironment

regtest coordinator on production != testnet coordinator on production
regtest backends != testnet backends

etc...

Reply:
yes, but coinjoinServerEnvironment is still just 'localhost' | 'public' or maybe 'dev' in a future, isn't it?